cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/logviewer/logparser/MockLogParserStrategy.kt

```diff
@@ -0,0 +1,29 @@
+package com.cognifide.cogboard.widget.type.logviewer.logparser
+
+import io.vertx.core.json.JsonObject
+import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.TYPE
+import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.DATE
+import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.PROVIDER
+import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.MESSAGE
+
+class MockLogParserStrategy : LogParserStrategy() {
```

Comment:
Nie wiem, czy MockLogParserStrategy to odpowiednia nazwa? Czy format logów, które tworzymy w logu trzyma się jakiejś konwencji, czy jest z czapy?

Reply:
Nie bardzo jestem w stanie znaleźć jakiej konwencji mógłby się trzymać ten format logów, stąd nazwa klasy. Dotyczy stricte tych logów, które są generowane w mocku